server/master/src/main/java/org/apache/accumulo/master/tableOps/TableRangeOp.java

```diff
@@ -38,19 +37,24 @@
   private static final long serialVersionUID = 1L;
 
   private final String tableId;
+  private final String namespaceId;
```

Comment:
I did some experiments with this and order does not seem to matter.  I tried reordering existing fields and inserting new fields and it did not matter.  Also I found some information that suggest the fields name is stored by serialization.